index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width initial-scale=1 user-scalable=no" />
+    <link href="http://fonts.googleapis.com/css?family=Lobster+Two:400700" rel="stylesheet" type="text/css">
+    <link href="http://fonts.googleapis.com/css?family=Open+Sans+Condensed:300700" rel="stylesheet" type="text/css">
```

Comment:
the font weights should be comma separated like `300,700`. All the regular text is bold now.